src/grandorgue/GrandOrgueFile.cpp

```diff
@@ -619,13 +619,16 @@ void GrandOrgueFile::LoadCombination(const wxString& file)
 		if (hash != wxEmptyString)
 			if (hash != m_ODFHash)
 			{
-				wxLogWarning(_("The combination file does not exactly match the current ODF."));
+				if (wxMessageBox(_("The combination file does not exactly match the current ODF. Importing it can cause various problems. Should it really be imported?"), _("Import"), wxYES_NO, NULL) == wxYES)
```

Comment:
Clicking every time to OK is not good.

Can you display the message box only if the churche name has changed (probably this combinations are from a totally another organ)? 

If the churche name is the same but hash is different then only to log warning?

Reply:
Sure, it's certainly possible. I'll fix that tomorrow!

Reply:
I've added a new solution that works better.